website/docs/docs/collaborate/govern/model-contracts.md

```diff
@@ -86,6 +86,85 @@ When building a model with a defined contract, dbt will do two things differentl
 1. dbt will run a "preflight" check to ensure that the model's query will return a set of columns with names and data types matching the ones you have defined. This check is agnostic to the order of columns specified in your model (SQL) or YAML spec.
 2. dbt will include the column names, data types, and constraints in the DDL statements it submits to the data platform, which will be enforced while building or updating the model's table.
 
+## Platform constraint support
+
+Adapter-specific constraint support across platforms. 
+
+<Tabs>
+
+<TabItem value="Redshift" label="Redshift">
+
+| constraint type | support       |
+|:----------------|:-------------|
+| not_null        | 🌕 supported & enforced     |
+| primary_key     | 🌗 supported; not enforced  |
+| foreign_key     | 🌗 supported; not enforced  |
+| unique          | 🌗 supported; not enforced  |
+| check           | 🌑 not supported  |
+
+</TabItem>
+<TabItem value="Snowflake" label="Snowflake">
+
+| constraint type | support       |
+|:----------------|:-------------|
+| not_null        | 🌕 supported & enforced     |
+| primary_key     | 🌗 supported; not enforced  |
+| foreign_key     | 🌗 supported; not enforced  |
+| unique          | 🌗 supported; not enforced  |
+| check           | 🌑 not supported |
+
+</TabItem>
+<TabItem value="BigQuery" label="BigQuery">
+
+| constraint type | support       |
+|:-----------------|:-------------|
+| not_null        | 🌕 supported & enforced     |
+| primary_key     | 🌕 supported & enforced     |
+| foreign_key     | 🌕 supported & enforced     |
+| unique          | 🌑 not supported |
+| check           | 🌑 not supported |
+
+</TabItem>
+<TabItem value="Postgres" label="Postgres">
+
+| constraint type | support       |
+|:----------------|:-------------|
+| not_null        | 🌕 supported & enforced     |
+| primary_key     | 🌕 supported & enforced  |
+| foreign_key     | 🌕 supported & enforced  |
+| unique          | 🌕 supported & enforced |
+| check           | 🌕 supported & enforced |
+
+</TabItem>
+<TabItem value="Spark" label="Spark">
+
+`not_null` and `check` constraints are supported and enforced. In the current implementation, however, these constraints are only enforced after a model builds. Due to this platform limitation, these are considered `supported` but `not enforced` and are not a part of the "model contract" since they can't be enforced at build time. This table will be updated as the features evolve.
```

Comment:
slight simplification
```suggestion
`not_null` and `check` constraints are supported and enforced. Currently,  constraints are enforced only after a model builds. Because of this platform limitation, the constraints are considered `supported` but `not enforced` and not part of the "model contract" since they can't be enforced at build time. This table will change as the features evolve.
```